_data/code/federal/cse-cst.yml

```diff
@@ -44,7 +44,7 @@ releases:
       metadataLastUpdated: '2019-01-31'
     description:
       en: Platform for the analysis of malicious files. It is designed to assist cyber defence teams to automate the analysis of files and to better use the time of security analysts
-      fr: Plateforme servant à l’analyse de fichiers malveillants, qui a été conçue pour aider les équipes de cyberdéfense à automatiser l’analyse des fichiers et à faire une meilleure utilisation du temps des analystes de la sécurité
```

Comment:
Changed apostrophe for single quote as it was causing an error with form validation